src/main/scala/code/snippet/ApiExplorer.scala

```diff
@@ -526,49 +526,10 @@ WIP to add comments on resource docs. This code copied from Sofit.
     )
 
 
-    // Filter
-    val filteredResources1: List[ResourceDocPlus] = showCore match {
```

Comment:
when those four request parameters send to obp-api, the follow filters already did in obp-api side, so here don't need do the same thing.